app/src/main/java/org/wikipedia/page/leadimages/LeadImagesHandler.kt

```diff
@@ -108,35 +109,39 @@ class LeadImagesHandler(private val parentFragment: PageFragment,
             }
             handlerJob = parentFragment.lifecycleScope.launch {
                 withContext(Dispatchers.Main) {
```

Comment:
We actually don't need a `withContext()` block here; it's already in the correct context.

Reply:
You are correct! I also changed `parentFragment.lifecycleScope` to `parentFragment.viewLifecycleOwner.lifecycleScope` to prevent crash.